tests/src/App.js

```diff
@@ -5,9 +5,61 @@ import histogram_data from './data/histogram.csv';
 import timeseries_data from './data/timeseries.csv';
 import xy_data from './data/xy.csv';
 import jitter_data from './data/jitter.csv';
+import dynamic_data_1 from './data/ts1.csv';
+import dynamic_data_2 from './data/ts2.csv';
+
+const d3 = require("d3");
```

Comment:
We need to include `d3` in the `package.json` file defined for the test environment via `yarn add d3`

Reply:
Ok.  I was wondering if you thought this was the best way to handle this or if should be done in a more react centric way as opposed to my d3 based approach.  Thoughts?

Reply:
Since this is being referenced in the React environment, we should approach it from React's conventions and most of the ES6 conventions that come with the environment. For example, using ES6 style import statements: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import

Reply:
Hmm, sorry I think I'm confused a bit.  I emulated the const call as you wrote it in index.js -- what is the difference between import (as in App.js) and const (as in index.js)?

I probably should have been more explicit with my previous question, I was referring to my usage of d3 in App.js (in the setup_dynamic_chart_selection_box function).

Reply:
Both methods work for importing modules; however, are written differently depending on the style of JavaScript used. Since our testing environment uses webpack and Babel for transpiling newer JavaScript syntax to "browser-compatible" JavaScript, we can load modules using ES6 style imports.

```
import d3 from 'd3'; //ES6 style import
const d3 = require("d3"); //Vanilla JS style import
```

These conventions for imports have a few benefits such as referencing multiple named exports from a module or creating aliases for named exports.